### PR TITLE
Add support for centos 7

### PR DESCRIPTION
**Pull request by cwood**

### CODE REVIEW

File: brokers/puppet.broker/install.erb [hunk 1]
```diff
@@ -52,6 +52,7 @@ case "${flavour}" in
         case "${release}" in
             5) url="http://yum.puppetlabs.com/el/5/products/i386/puppetlabs-release-5-7.noarch.rpm" ;;
             6) url="http://yum.puppetlabs.com/el/6/products/i386/puppetlabs-release-6-7.noarch.rpm" ;;
+            7) url="ttps://yum.puppetlabs.com/puppetlabs-release-el-7.noarch.rpm" ;;
```
Comment (smcclellan) [inline brokers/puppet.broker/install.erb]: Missing an `h` at the start of this, otherwise looks good.

Comment (cwood) [inline brokers/puppet.broker/install.erb]: My bad just used the github editor

**Comment by smcclellan** [comment]
Have you signed the CLA at https://cla.puppetlabs.com?

**Comment by cwood** [comment]
Just did

**Comment by smcclellan** [comment]
Squashed commits and merged as in commit d745d3093291ae908dfcbdfc1660d67f968ce242.

**Pull request closed without merging**